<launch>
    <group ns="steer">
        <node pkg="joy" type="joy_node" name="joy_node">
            <param name="default_trig_val" type="bool" value="true" />
            <!-- <param name="dev" type="str" value="/dev/input/js0" /> -->
        </node>
        
        <node pkg="steering" type="Circadian.py" name="circadian">
            <param name="pulseRate" type="int" value="1000" />
        </node>

        <node pkg="steering" type="Steering.py" name="steering">
            <param name="cmd_topic" type="str" value="/cmd_vel" />
            <param name="max_lin_vel" type="double" value="1.5" />
            <param name="max_rot_vel" type="double" value="0.461" />
        </node>
    </group>

    <remap from="/cam_node/image_raw/compressed" to="/steer/cam_feed" />
</launch>